packages/ckeditor5-image/src/imageinsert/utils.ts

```diff
@@ -17,11 +19,11 @@ import { LabeledFieldView, createLabeledInputText } from 'ckeditor5/src/ui';
  *
  * @returns {Object.<String, module:ui/view~View>} Integrations object.
```

Comment:
Remember to strip types in JSDoc comments. I can see plenty of those left in many files. Please find them all and remove.